tests/unit_tests/libs/controllers/test_predictor.py

```diff
@@ -205,7 +258,7 @@ def test_predictor_deduplicate_data(self):
 
         model_data = F.get_model_data('test_drop_duplicates')
 
-        # Ensure duplicate row was not used for training
+        # Duplicate row was used for analysis and training
 
         assert model_data['data_preparation']['total_row_count'] == n_points+1
```

Comment:
why is it `n_points + 1`?

Reply:
@maximlopin there were `n_points` generated, and then one was duplicated, so there are `n_points+1` in the data frame.

This is the second part of the test. It checks that if deduplication is disabled the duplicated row is used for operations.